script.py

```diff
@@ -3,12 +3,12 @@
 
 
 def get_nightly_version():
-    r = requests.get('https://product-details.mozilla.org/1.0/firefox_versions.json')
-    return r.json()['FIREFOX_NIGHTLY']
+    return '68.0a1'
```

Comment:
We could use `alpha_version` from https://product-details.mozilla.org/1.0/mobile_versions.json

Reply:
I considered that.  We'll need to update the download URL in a little while anyway when moving from beta to esr68, and I'm not sure what the version will be after esr68 branches out, so I'll keep this in mind for a later update.